Nancy.Rollbar/Properties/AssemblyInfo.cs

```diff
@@ -30,5 +30,5 @@
 //
 // You can specify all the values or you can default the Revision and Build Numbers 
 // by using the '*' as shown below:
-[assembly: AssemblyVersion("1.2.0.0")]
-[assembly: AssemblyFileVersion("1.2.0.0")]
+[assembly: AssemblyVersion("1.3.0.0")]
+[assembly: AssemblyFileVersion("1.3.0.0")]

```

Comment:
We should be using semantic versioning conventions here. The change you've introduced is a pretty straightforward bug fix which maintains backwards compatibility, so I think we ought to be bumping the version to 1.2.1 rather than 1.3.

[See Semantic Versioning](http://semver.org/)